backend/templates/home.html

```diff
@@ -1,44 +1,55 @@
-                The Federal Audit Clearinghouse (FAC) is the place to submit and review federal
-                grant audits.
-            </h1>
-            <p class="font-sans-lg">
-                When an organization spends $750,000 or more in federal grant funds in a given year, they are required to
-                submit an audit.
-            </p>
-            <ul class="usa-card-group margin-top-10">
-                <li class="usa-card desktop:grid-col-6">
-                    <div class="usa-card__container">
-                        <div>
-                            <a class="usa-button sign-in-button"
-                               aria-controls="login-modal"
-                               data-open-modal
-                               href="/openid/login/">Sign in</a>
-                        </div>
-                        <p class="usa-card__body">Sign in to submit and review your audits.</p>
-                    </div>
-                </li>
-                <li class="usa-card desktop:grid-col-6">
-                    <div class="usa-card__container">
-                        <div>
-                            <a class="usa-button usa-button--outline"
-                               href="{% url 'dissemination:Search' %}">Search audit reports</a>
-                        </div>
-                        <p class="usa-card__body">
-                            Search audits accepted since October 1, 2023.
-                            Older reports are available
-                            <a class="usa-link usa-link--external"
-                               href="https://facides.census.gov/Account/Login.aspx"
-                               target="_blank">from the Census Bureau</a>
-                            .
-                        </p>
-                    </div>
-                </li>
-            </ul>
-        </div>
-    </section>
+<div class="home margin-bottom-4">
+    <div class="grid-container">
+      {# Title #}
+      <h1 class="font-sans-3xl text-primary-darker">
+          <span class="text-heavy">
+              The Federal Audit Clearinghouse (FAC)
+          </span>
+          <span class="text-semibold">
+              is the place to submit and review federal grant audits.
+          </span>
+      </h1>
+  
+      <hr />
+  
+      {# Cards #}
+      <ul class="usa-list usa-list--unstyled width-full margin-top-4">
+          <li class="usa-card">
+              <div class="usa-card__container radius-0 border-2px border-base-light">
+                  <div class="usa-card__header border-bottom-2px border-base-light bg-accent-cool-lighter padding-y-105">
+                      <h2 class="usa-card__heading font-sans-2xl text-primary-darker text-bold">I want to</h2>
+                  </div>
+                  <div class="usa-card__body bg-base-lightest margin-0">
+                      <ul class="usa-card-group display-flex flex-row flex-justify">
```

Comment:
Not sure if this would look different on a physical device, but when I use the Chrome device simulator, these cards get squished and the text overruns them: 

<img width="355" alt="image" src="https://github.com/GSA-TTS/FAC/assets/2365503/23e592de-0626-4453-97d5-33ee8800c5af">

Reply:
Thanks for the catch! Went ahead and made the `flex-row` desktop only. I also applied it to the static side homepage as well. Now, mobile users should see something akin to:

![image](https://github.com/GSA-TTS/FAC/assets/91098850/e1ec8d4c-1647-421d-a8da-f6b55548f4cb)

Reply:
Tightened the margins on the 'or' as well, to reduce whitespace.

![image](https://github.com/GSA-TTS/FAC/assets/91098850/492973ce-e6ff-4d5d-9121-a0fed008a472)